website/docs/artist_details_panel.md

```diff
@@ -0,0 +1,40 @@
+---
+id: artist_details_panel
+title: The Task Details Panel
+sidebar_label: Details Panel
+---
+
+:::info
+The details panels received updates in server version `1.3.0`.
+:::
+
+## Overview 
+When using the AYON production tracker, you'll encounter the details panel frequently. It's essential for team interactions.
+As the name suggests, the details panel is packed with information about tasks, serving as a communication tool and offering useful actions.
+
```

Comment:
```suggestion
As the name suggests, the details panel is packed with information, serving as a communication tool and offering useful actions.
```

Reply:
Thank you. 
https://github.com/ynput/ayon-documentation/pull/287/commits/069be35feeac7fabad542741b1bfa6d7643d8195